### PR TITLE
Rp/checkpoint 1

### PR DESCRIPTION
**Pull request by RyanPearson51**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+}
+
+.scpic{
+    width: 90%;
+    height: 200px;
+    border: 5px solid violet;
+}
+
+
+.tc1{
```
Comment (juan-rod) [inline style.css:45]: i'd be a little more descriptive when creating class names. it helps when other people are reading your code!

File: news.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                <li><a href = "./index.html">Home</a></li>
+                <li><a href = "./about.html">About</a></li>
+                <li><a href = "./gallery.html">Gallery</a></li>
+                <li><a href = "./news.html">News</a></li>
+                <li><a href = "./contact.html">Contact</a></li>
+            </ul>
+            </nav>
+          </header>
+
+          <div class = "container">
```
Comment (juan-rod) [inline news.html:27]: no spaces between class, equals sign, and quotes. all attributes should look like this:
 `<div class="container">`

File: index.html [hunk 3]
```diff
@@ -7,24 +7,76 @@
     <title>Hair Salon Checkpoint</title>
     <link rel="stylesheet" href="./reset.css">
     <link rel="stylesheet" href="./style.css">
+    <link rel = "stylesheet" href = "./nav.css">
```
Comment (juan-rod) [inline index.html:10]: no space for the attributes. you want to match line 8 and 9. where the equals sign is touching the attribute name and value. javascript allows spaces. you can get away with it in html but it's not the norm.

**Review by juan-rod — APPROVED** [review]
great job! just a few comments but no big issues! what questions do you have? let me know! 
merge to main when ready.

**Comment by RyanPearson51** [comment]
Hey Juan, thanks for the comments. I think the biggest question I have for future assignments is about the responsiveness.  Obviously the way I did it I was able to make it work starting from the desktop view and adding media queries for smaller versions, but ive seen that usually its easier to start with mobile and then build from there.  In the future do you recommend one way or the other or is it fine either way?

**Comment by juan-rod** [comment]
yeah. that's hard to define because everyone works a little different. when i think about creating a site, i always design the desktop version first and then i jump in and work on the desktop version. But i do think I have to spend more time getting mobile to work. I'd say do what is most efficient for you!

**Pull request merged**